src/Services/QueuedJobService.php

```diff
@@ -493,6 +524,10 @@ public function checkJobHealth($queue = null)
             $job->write();
         }
 
+        // Attempt to automatically retry eligible jobs before reporting any broken jobs
+        // as some of these jobs might be switched from broken state
+        $this->retryEligibleJobs($queue);
```

Comment:
```suggestion
        // Mark eligible jobs to be retried before reporting any broken jobs
        // as some of these jobs might be switched from broken state
        $this->retryEligibleJobs($queue);
```
Could use the word "Schedule" instead of "Mark" depending on which word is used in the method name.